.github/release.yml

```diff
@@ -0,0 +1,22 @@
+# .github/release.yml
+
+changelog:
```

Comment:
How does this work exactly?  Does this update a github release somewhere?  We haven't been doing github releases for the prerelease version of the extension (though we could).

Reply:
yes this will hook up to the github release note generation with the new labels where we can set the release to be pre-release until GA.